src/components/Pricing/Addons/index.tsx

```diff
@@ -39,12 +39,17 @@ export const Addons = ({ billingProducts }: AddonsProps) => {
                 return product.addons
             })
             .flat()
-            .filter((addon: any) => !addon.inclusion_only)
     }, [billingProducts])
 
     const getAddonPrice = (addon: any): number | null => {
-        const tiers = addon.plans[0]?.tiers
-        if (tiers.length > 0) {
+        const isInclusionOnly = addon.inclusion_only
+        let tiers
+        if (isInclusionOnly) {
+            tiers = addon.plans.find((plan) => plan.included_if == 'has_parent_subscription')?.tiers
```

Comment:
Maybe add a comment as to why we're doing this because it's not immediately clear. The `included_if` stuff is hard to follow.